renderer/components/home/running.js

```diff
@@ -158,10 +156,16 @@ const RunningTest = ({testGroup, logOpen, onToggleLog, progressLine, percent, lo
         trailColor='rgba(255,255,255,0.4)'
         trailWidth='2'
       />
-      <Text>
-        <FormattedMessage id='Dashboard.Running.EstimatedTimeLeft' />
-        <FormattedMessage id='Dashboard.Running.Seconds' values={{ seconds: eta }}/>
-      </Text>
+      {eta > 0 &&
+      <Flex justifyContent='center'>
+        <Box pr={1}>
+          <FormattedMessage id='Dashboard.Running.EstimatedTimeLeft' />
+        </Box>
+        <Box>
+          <FormattedMessage id='Dashboard.Running.Seconds' values={{ seconds: moment.duration(eta*1000).humanize() }}/>
```

Comment:
By using `moment.duration.humanize()` we would force strings like 'seconds' and 'minutes' in non-en locales. One solution is to set a locale for the `moment` instance like [here](https://momentjs.com/docs/#/durations/locale/).